lib/routes/dedao/articles.ts

```diff
@@ -0,0 +1,149 @@
+import { Route } from '@/types';
+import cache from '@/utils/cache';
+import got from '@/utils/got';
+import { load } from 'cheerio';
+
+export const route: Route = {
+    path: '/articles/:id?',
+    categories: ['new-media'],
+    example: '/articles/9', // 示例路径更新
+    parameters: { id: '文章类型 ID，8 为得到头条，9 为得到精选，默认为 8' },
+    features: {
+        requireConfig: false,
+        requirePuppeteer: false,
+        antiCrawler: false,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    radar: [
+        {
+            source: ['igetget.com'],
+            target: '/articles/:id',
+        },
+    ],
+    name: '得到文章',
+    maintainers: ['qi12371'],
```

Comment:
This user [does not exist](https://github.com/qi12371).
```suggestion
    maintainers: ['Jacky-Chen-Pro'],
```

Reply:
Sorry, this is my username on another platform. I have changed my GitHub name now.

Reply:
In fact, this is my first time submitting a pull request on GitHub. Thank you for your help!